kapacitor/templates/tickscript-init-job.yaml

```diff
@@ -0,0 +1,74 @@
+{{if .Values.install_tick_scripts.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-init
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  init.sh: |-
+    #!/usr/bin/env bash
+    export KAPACITOR_URL={{ .Values.install_tick_scripts.kapacitorURL }}
+    ls -al $TICK_DIR
+    kapacitor list tasks
+
+    echo "Loading TICK files from $TICK_DIR"
+    for f in ${TICK_DIR}/*.tick
+    do
+        name="$(basename $f)"
+        name="${name/.tick/}"
+        kapacitor define $name \
+            -type batch \
+            -dbrp telegraf.autogen \
```

Comment:
For my purposes I don't (yet) need it to vary, which is why I hardcoded it. I figured I could bake it into the dir structure if I needed to parameterize it someday, e.g. `$TICK_DIR/telegraf/autogen/foo.tick`. If you want to bake it into the TICK script itself and parse it out in kapacitor, sounds good to me 😁.